tests/unittests/CoreText/CTLineTests.mm

```diff
@@ -98,4 +98,141 @@
     // comparing secondaryOffset and offset.
     offset = CTLineGetOffsetForStringIndex(line, 4, &secOffset);
     ASSERT_EQ_MSG(offset, secOffset, "Failed: Wrong offset for given index");
+}
+
+TEST(CTLine, CTLineCreateWithAttributedString) {
+    NSMutableAttributedString* attrString = getAttributedString(@"hello");
+    [attrString addAttribute:NSFontAttributeName value:[UIFont systemFontOfSize:20] range:NSMakeRange(3, 2)];
+    CFAttributedStringRef string = (__bridge CFAttributedStringRef)attrString;
+
+    CTLineRef line = CTLineCreateWithAttributedString(string);
+    EXPECT_NE(line, nil);
+
+    EXPECT_EQ(0, CTLineGetStringRange(line).location);
+    EXPECT_EQ(5, CTLineGetStringRange(line).length);
+    EXPECT_EQ(5, CTLineGetGlyphCount(line));
+    CFArrayRef runs = CTLineGetGlyphRuns(line);
+
+    EXPECT_NE(runs, nil);
+    EXPECT_EQ_MSG(2, CFArrayGetCount(runs), "CTLine should create multiple runs when attributed string has multiple attribute ranges");
+    EXPECT_EQ(3, CTRunGetGlyphCount(static_cast<CTRunRef>(CFArrayGetValueAtIndex(runs, 0))));
+    EXPECT_EQ(2, CTRunGetGlyphCount(static_cast<CTRunRef>(CFArrayGetValueAtIndex(runs, 1))));
+
+    // Glyphs for h, e, and l respectively
+    const CGGlyph* helGlyphs = CTRunGetGlyphsPtr(static_cast<CTRunRef>(CFArrayGetValueAtIndex(runs, 0)));
+    ASSERT_NE(nullptr, helGlyphs);
+    EXPECT_EQ(75, *helGlyphs);
```

Comment:
>EXPECT_EQ(75, *helGlyphs); [](start = 4, length = 26)

This test will fail if we change our default font or the locale. Might be a good idea to set the font name and locale for this test for such hard coded values